network/router.go

```diff
@@ -197,7 +220,7 @@ func (r *Router) handleConn(remote *ServerIdentity, c Conn) {
 		if err != nil {
 			log.Lvlf4("%+v got error (%+s) while receiving message", r.ServerIdentity.String(), err)
 
-			if err == ErrClosed || err == ErrEOF {
+			if err == ErrClosed || err == ErrEOF || err == ErrTimeout {
```

Comment:
Now I have a lot of `Error: timeout` in my logs. Can we put that in a separate `if`-clause and not output an error for the `ErrTimeout`?